packages/path_provider/path_provider_macos/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.3+1
```

Comment:
This should be 0.0.3. We only add +x when it's a change that doesn't affect the API or actual code (readme updates, comments etc)